cmd/cleanup.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Please remove the `cleanup` command from this PR.

Reply:
Done.